openstack/octavia/templates/octavia-api-deployment.yaml

```diff
@@ -34,7 +34,7 @@ spec:
         prometheus.io/targets: {{ required ".Values.alerts.prometheus missing" .Values.alerts.prometheus | quote }}
         {{- end }}
     spec:
-      {{- include "utils.proxysql.pod_settings" . | nindent 6 }}
```

Comment:
That's intentional

`nindent` adds a newline before indenting `indent` doesn't.

Reply:
Fixed in [new commit](https://github.com/sapcc/helm-charts/commit/57866ddcac7a15d8c18fe83c5192afc8060853ef)